packages/clay-css/src/scss/cadmin/components/_navbar.scss

```diff
@@ -65,6 +65,7 @@
 		align-items: center;
 		display: flex;
 		word-wrap: normal;
+		max-width: 100%;
```

Comment:
Hey @pat270 I added this in `nav-item` because when the text is too big even if it has the truncate class when the navigation-bar is in mobile version it doesn't respect the screen size, let me know if this can break some cases of use.

Reply:
I talked to Patrick about it, it's ok to go with it.